cf-execd/cf-execd-runner.c

```diff
@@ -713,30 +735,33 @@ static void MailResult(const ExecConfig *config, const char *file)
     send(sd, vbuff, strlen(vbuff), 0);
 
     int count = 0;
-    while (!feof(fp))
+    while ((read = CfReadLine(&line, &line_size, fp)) > 0)
     {
-        vbuff[0] = '\0';
-        if (fgets(vbuff, sizeof(vbuff), fp) == NULL)
+        if (LineIsFiltered(config, line))
         {
-            break;
+            continue;
         }
-
-        Log(LOG_LEVEL_DEBUG, "Mail report: %s", vbuff);
-
-        if (strlen(vbuff) > 0)
+        if (send(sd, line, read, 0) == -1 ||
+            send(sd, "\r\n", 2, 0) == -1)
```

Comment:
Indeed, but this was a bug. The RFC clearly states that lines should be separated by CRLF.